lib/OpenQA/Worker/Jobs.pm

```diff
@@ -21,6 +21,7 @@ use feature 'state';
 use OpenQA::Worker::Common;
 use OpenQA::Worker::Pool qw/clean_pool/;
 use OpenQA::Worker::Engines::isotovideo;
+use OpenQA::Utils qw/wait_with_progress/;
```

Comment:
please use qw as function in new code - and change the surrounded code to do the same :)